spring-cloud-stream-tuple/src/main/java/org/springframework/cloud/stream/tuple/JsonNodeToTupleConverter.java

```diff
@@ -66,6 +75,15 @@ else if (node.isArray()) {
 			else if (item.isArray()) {
 				list.add(nodeToList(item));
 			}
+			else if (node.isNull()) {
```

Comment:
Do these checks belong here? AFAICT it's already been established that the node is an array.